backend/src/routes/api/status/index.ts

```diff
@@ -9,8 +9,15 @@ const status = async (
   request: FastifyRequest,
 ): Promise<{ kube: KubeStatus }> => {
   const kubeContext = fastify.kube.currentContext;
-  const { currentContext, namespace, currentUser } = fastify.kube;
-  const userName = request.headers['x-forwarded-user'] ?? currentUser.username;
+  const { currentContext, namespace, currentUser, customObjectsApi } = fastify.kube;
+  const userName = request.headers['x-forwarded-user'] ?? currentUser.name;
+  const clusterVersion = await customObjectsApi.getClusterCustomObject(
+    'config.openshift.io',
+    'v1',
+    'clusterversions',
+    'version',
+  );
```

Comment:
This should be done once at startup in `kube.fp` and added to the kube decorator.